src/Nest/Search/Suggesters/CompletionSuggester/FuzzySuggest.cs

```diff
@@ -3,35 +3,61 @@
 
 namespace Nest
 {
+	/// <summary>
```

Comment:
`FuzzySugester` is a bad name now that I am rereading it maybe `SuggesterFuzziness`?

Maybe link to https://www.elastic.co/guide/en/elasticsearch/reference/current/search-suggesters-completion.html#fuzzy ?

The comment itself as it is now adds little IMO.

Reply:
++ to `SuggesterFuzziness`; it better fits what it is.

I share some of the sentiment of GhostDoc/Sandcastle autogenerated XML comments. Do you think we should aim for consistency in applying `<summary>`?

Reply:
Looking at the naming convention of other types, `ISuggestFuzziness` might be more appropriate